ansible-tests/validations/controller-ulimits-check.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: controller
+  vars:
+    metadata:
+      name: Check controller ulimits
+      description: >
+        This will check the ulimits of each controller.
+      groups:
+      - post-deployment
+    nofiles_default: 1025
+    nproc_default: 1025
+  tasks:
+  - name: Get nofiles limit
+    become: true
+    command: sh -c "ulimit -n"
```

Comment:
In testing, I had to do this since 'ulimit' is a shell built in command.  Doing it without wrapping it causes a failure.